views/continue-trading.html

```diff
@@ -0,0 +1,29 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Tell us if a company is still required
+{% endblock %}
+
+{% block header %}
```

Comment:
blockHeader is in layout.html - does this page require a different header to the one in the layout?

Reply:
this is a dummy page until story for this is picked up.